seq2science/rules/get_genome.smk

```diff
@@ -32,7 +32,7 @@ rule get_genome_annotation:
     Download a gene annotation through genomepy.
     """
     input:
-        ancient(rules.get_genome.output),
+        rules.get_genome.output,
```

Comment:
still causing issues? it seemed to work in my trial runs...

Reply:
Nope, I guess that stayed from the merge from the update_snakemake branch from before, sloppy